documentation/modules/exploit/unix/webapp/wp_infinitewp_auth_bypass.md

```diff
@@ -0,0 +1,80 @@
+## Introduction
```

Comment:
Vulnerable Application.
If you still want Introduction (inside of the Vulnerable Application h2), I believe an H3 is in order.

Reply:
Don't need it.

Reply:
Preferring `### Description` now.